### PR TITLE
Fix missing link style on "basic camera mode" error

### PR DESCRIPTION
**Pull request by kopijunkie**
# Problem
When going through the SDK's verification flow on a mobile with getUserMedia API support, on the selfie/liveness capture step if a user doesn't take a photo/video before the 10s timeout the "Camera not working" error message gets displayed but there is no obvious link to tap on (i.e. the "basic camera mode" line) 

Screenshot of error message bug

# Solution
All error/warning messages should have a consistent style with underlined text style for interactive elements that are not the Close button, if present on the message.

Screenshot of error message with link style

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [x] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://857-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Capture/Face.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ class Face extends Component {
   }
 
   renderUploadFallback = text =>
-    <CustomFileInput onChange={this.handleUpload} accept="image/*" capture="user">
+    <CustomFileInput className={style.faceCaptureFallback} onChange={this.handleUpload} accept="image/*" capture="user">
```
Comment (stefaniacardenas) [inline src/components/Capture/Face.js]: Does the document live capture have the same bug? In that case we should fix it for that component too

Comment (kopijunkie) [reply src/components/Capture/Face.js]: Good point. Will also update the acceptance criteria in the ticket so it gets checked also.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-857-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://857-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**